app/main.py

```diff
@@ -1,9 +1,6 @@
-from typing import Union
+class Distance():
```

Comment:
$1 i think you don't need ( ) here
$21 it's better to use operator "+="
$29 remove pass

Reply:
thanks , i fix it